<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuideDialog</class>
 <widget class="QDialog" name="GuideDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_mask">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>757</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/Share/images/GuideDialogHint.PNG);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_showUp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>505</y>
     <width>161</width>
     <height>19</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(209, 209, 209);</string>
   </property>
   <property name="text">
    <string>不再显示启动向导</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_close">
   <property name="geometry">
    <rect>
     <x>677</x>
     <y>495</y>
     <width>90</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_getStart">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>启动向导</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_learn">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>115</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>从这里开始学习使用 Cell 深度学习系统</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
